### ISSUE TITLE
Multiple file upload

### ISSUE THREAD

**Issue by mohsinmarui** [waifung0207/ci_bootstrap_3#34]
Hello Dear,

you have created awesome CI3 admin panel. i am using your CI admin panel Thank you for creating and share as publicly.
Is there multiple file upload code in that CI3 admin panel ? if yes how do i check it

Thanks in Advance

**Comment by waifung0207** [comment]
Yes you can add multiple fields in Grocery CRUD, simply check its official doc about set_field_upload() function here: http://www.grocerycrud.com/documentation/options_functions/set_field_upload

You can target multiple fields to different directories - just make sure the destinations are writable.

Or if you require multiple image upload, you can make use of Image CRUD library instead, which I have put some sample code inside /application/modules/admin/controllers/Demo.php > cover_photo(); you will see a shortcut function named generate_image_crud() is called, and its logic is located in /application/core/controllers/Admin_controller.php so you will know what had happened there.
